### PR TITLE
Added new polyfeed endpoints

### PR DESCRIPTION
**Pull request by julio-santiesteban**
with linter fix

### CODE REVIEW

**Comment by julio-santiesteban** [comment]
check

**Comment by julio-santiesteban** [comment]
Why is the CI not running for this PR?

File: README.md [hunk 1]
```diff
@@ -163,6 +163,18 @@ of `1Min`. Similarly, `day` is an alias of `1D`.
 `pd.Timestamp().isoformat()`
 `after` cannot be used with `start` and `until` cannot be used with `end`.
 
+### REST.get_aggregates(symbol, timespan, multiplier, _from, to):
```
Comment (umitanuki) [inline README.md:166]: The parameter order is not consistent.

Comment (gnvk) [reply README.md:166]: good catch, fixed

**Pull request merged**